string.js

```diff
@@ -0,0 +1,103 @@
+//  1. `console.log` the first character of a string.
+ let string1= 'garcia'
+ console.log(string1[0])
+//
+//  2. `console.log` the length of a string.
+console.log(string1.length)
+//
+//  3. `console.log` the last character of any string.
+ console.log(string1[string1.length-1])
+
+//  4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+ let str3 = "juan"
+ console.log(str3.toUpperCase(str3[str3.length-1])) // capitalizes all characters.
+console.log('------------------')
+ let name= 'juan'
+ let capitalized = name.slice(name.length) + name[name.length-1].toUpperCase()
```

Comment:
You got the right idea here of using slice method but (name.slice(0, 3)) would not work if name was set to "Alejandro" instead. How can we fix this problem? This is instance of hard coding which is when different input gives wrong output or breaks the code. Try to avoid this.